docs/podman.1.md

```diff
@@ -171,6 +171,17 @@ Containers created by a non-root user are not visible to other users and are not
 
 It is required to have multiple uids/gids set for an user.  Be sure the user is present in the files `/etc/subuid` and `/etc/subgid`.
 
+If you have a recent version of usermod, you can execute the following
+commands to add the ranges to the files
+
+	$ sudo usermod --add-subuids 10000-65536 USERNAME
```

Comment:
I checked and it adds less users than the manual command below (the second number is the maximum ID, not the length of the range).  Let's change it to `10000-75535`